### ISSUE TITLE
[Bug]: Some accentuated words don't get suggested corrections.

### ISSUE THREAD

**Issue by gojeaqui** [streetsidesoftware/vscode-spell-checker#4761]
### Kind of Issue

Change in behavior / Regression

### Tool or Library

cspell

### Version

4.2.6 / 2.3.9 (spanish)

### OS

Windows

### OS Version

Windows 10

### Description

I have been dealing with this issue that some words have no suggestions despite the fact that the word is in fact in the dictionary. I only get English suggestions.

I usually have the English and Spanish dictionaries enabled because I create technical documentation and it is very bilingual.

<img width="624" height="324" alt="Image" src="https://github.com/user-attachments/assets/f47c7aeb-d10f-4ecf-8c49-369134803ff1" />

In the example shown the word "preparacion" has only an English suggestion, when there is an easy fix in Spanish: preparación

It might seem that the (preferred) suggestion in English is trumping other suggestions...

### Steps to Reproduce

Enable both English and Spanish dictionaries.

Type: preparacion, the word should be highlighted for correction but only suggestion is: "preparation", no suggestions in Spanish 

### Expected Behavior

There should be suggested corrections for Spanish

### cspell.json

```jsonc

```

### cspell.config.yaml

```yml

```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by gojeaqui** [comment]
Update: I just found out that `(right click) -> Spelling Suggestions...` *does* suggest the corrections.

Another thing I noticed:
<img width="311" height="87" alt="Image" src="https://github.com/user-attachments/assets/4a4bd3dc-dbab-4366-946b-9e4b68d545d9" />

Versus:
<img width="299" height="107" alt="Image" src="https://github.com/user-attachments/assets/f21428a1-7c25-4aaa-ad72-5d5f96f379fe" />

"Misspelled word" versus "Unknown word" ("Misspelled word" gets only suggestions in English).

And another difference is that the word "politicas" has a blue light bulb for the correction, while the word "politica" has a yellow light bulb...

**Comment by nschonni** [comment]
Guessing this is something to transfer to https://github.com/streetsidesoftware/vscode-spell-checker

**Comment by gojeaqui** [comment]
Thanks @nschonni I didn't realize they were different projects.

**Comment by Jason3S** [comment]
@gojeaqui,

Thank you for raising the issue.

It is working as "designed", but it clearly is not helpful when two languages are used.

The spell checker uses a "typos" dictionary to make suggestions for commonly misspelled words. The current behavior is that when a known typo is found, only return the given suggestions and do not look for other possible matches. Since the only "typos" dictionary available is in English, no Spanish suggestions are given if the misspelled word is a common English misspelling. 

The fix is to return all suggestions.

> Another thing I noticed: 
> <img alt="Image" width="311" height="87" src="https://private-user-images.githubusercontent.com/58303431/488414919-4a4bd3dc-dbab-4366-946b-9e4b68d545d9.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgwOTM5NzUsIm5iZiI6MTc1ODA5MzY3NSwicGF0aCI6Ii81ODMwMzQzMS80ODg0MTQ5MTktNGE0YmQzZGMtZGJhYi00MzY2LTk0NmItOWU0YjY4ZDU0NWQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTE3VDA3MjExNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY5ZGU4ODdiNWU1ZmJmODU5OGE0OWU1MDgwYTRlZTE0ZTk1MDk3ZDFkZTBjOTZiMTYyNmYwYzhiMTUyMTUxNjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mQi2YDRRkr0sS0j7hNYvWPzWJkWuPsxGkh6QGf_b_4A">
> 
> Versus: 
> <img alt="Image" width="299" height="107" src="https://private-user-images.githubusercontent.com/58303431/488415058-f21428a1-7c25-4aaa-ad72-5d5f96f379fe.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgwOTM5NzUsIm5iZiI6MTc1ODA5MzY3NSwicGF0aCI6Ii81ODMwMzQzMS80ODg0MTUwNTgtZjIxNDI4YTEtN2MyNS00YWFhLWFkNzItNWQ1Zjk2ZjM3OWZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTE3VDA3MjExNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTljYWU2ZmE4ZTY4OTcyZGRhM2ZmOWM2MzUwZWM1NzQwMjI3N2NhMzJhZTc1YTY5M2E2MjFjMDRmNGUzZWYxN2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.kVImMuPwHzyg6usNYf8a-O7wdL0MKWdXXi-hp9MDUH8">
> 
> "Misspelled word" versus "Unknown word" ("Misspelled word" gets only suggestions in English).


- **`Misspelled word`** - means the word was found in the typos dictionary.
- **`Unknown word`** - means the word was not found in any dictionary.

**Comment by gojeaqui** [comment]
> The current behavior is that when a known typo is found, only return the given suggestions and do not look for other possible matches

How about an option to change that behaviour. Because at some point I was actually looking for an option to disable the suggestions because that seemed to be the problem for me.
